docs/source/tutorials/low-latency.rst

```diff
@@ -120,6 +120,30 @@ Reducing latency when running NN
 In the examples above we were only streaming frames, without doing anything else on the OAK camera. This section will focus
 on how to reduce latency when also running NN model on the OAK.
 
+Resource utilization
+--------------------
+
+Configuring `hardware resources <https://docs.luxonis.com/projects/hardware/en/latest/pages/rvc/rvc2.html#hardware-blocks-and-accelerators>`__
+on RVC will result in lower latency, but also in lower FPS.
+
+By default, NN nodes have run 2 threads and 1 NCE/thread, and it's suggested to compile the model for half of the
```

Comment:
Maybe mention why. Above 5/6 shaves the improvement is marginal, due to:
1. Model optimizer
2. HW, paralellizing operations (splitting operations in a layer and assigning to multiple shaves) doesn't scale well due to  "[memory wall](https://en.wikipedia.org/wiki/Random-access_memory#Memory_wall:~:text=speed.%5B31%5D-,Memory%20wall,-The%20%22memory%20wall)" etc.

That's why running 2 threads with half the available total shaves for NN and 1 NCE per thread is the most optimal for throughput, since each thread can run freely.